tests/README.md

```diff
@@ -0,0 +1,81 @@
+## Backend development
+
+The backend test suite can be run locally. 
+
+### Start the services
+
+We need all but the main `backend` service running.  Since multiple `backend` instances are allowed, we will simply start everything up
+
+    # ENSURE THE IMAGES ARE CLEAN
+    docker-compose down
+    # SETUP ALL IMAGES
+    docker-compose up --build
+
+### Install into virtual environment
+
+Treeherder, requires specific library versions that will likely interfere with with what you have installed. It is best to setup a virtual environment to contain the quirks it introduces.
+
+Be sure you are in the `treeherder` main directory
+
+    pip install virtualenv
+    virtualenv venv
+    source venv/bin/activate
+    venv/bin/pip install -r requirements/common.txt
+    venv/bin/pip install -r requirements/dev.txt
+    venv/bin/pip install -r requirements/docs.txt
+
+...or Windows...
+
+    pip install virtualenv
+    virtualenv venv
+    venv\Scripts\activate
+    python -m pip install -r requirements\common.txt
+    python -m pip install -r requirements\dev.txt
+    python -m pip install -r requirements\docs.txt  
+
+### Set environment variables
+
+Treeherder requires a number of environment variables that point to the services.  In our case, those services are in local docker containers.
```

Comment:
I think we could change `treeherder/config/settings.py` to set some of this up for you if you're running tests.
Since it is so late in the life of this PR feel free to punt it.